### PR TITLE
[2.10] update exporting metrics docs

### PR DESCRIPTION
**Pull request by Pothulapati**
This PR updates the `Exporting Metrics` doc to reflect the changes
that happened with 2.10

Signed-off-by: Tarun Pothulapati <tarunpothulapati@outlook.com>

### CODE REVIEW

**Review by dadjeibaah — APPROVED** [review]
📦 🚢

File: linkerd.io/content/2/tasks/exporting-metrics.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ Similar to the `/federate` API, Prometheus provides a JSON query API to
 retrieve all metrics:
 
 ```bash
-curl http://linkerd-prometheus.linkerd.svc.cluster.local:9090/api/v1/query?query=request_total
+curl http://prometheus.linkerd-viz.svc.cluster.local:9090/api/v1/query?query=request_total
 ```
 
 ## Gathering data from the Linkerd proxies directly {#proxy}
```
Comment (adleong) [inline linkerd.io/content/2/tasks/exporting-metrics.md:141]: Should we mention the `linkerd diagnostics proxy-metrics` command?

**Review by adleong — COMMENTED** [review]
I think we should edit the introduction at the start of this page to give the proper context that Promtheus is part of the Viz extension and the instructions on this page requiring having the viz extension installed

File: linkerd.io/content/2/tasks/exporting-metrics.md (hunk 1 above)
Comment (Pothulapati) [reply linkerd.io/content/2/tasks/exporting-metrics.md:142]: Definitely makes sense to add this.

**Comment by Pothulapati** [comment]
@adleong Updated.

**Review by alpeb — APPROVED** [review]
LGTM :+1:

**Pull request merged**